src/Controller/MainController.php

```diff
@@ -349,7 +352,8 @@ public function tx($hash = null) {
         }
 
         $block = $this->Blocks->find()->select(['confirmations', 'height'])->where(['hash' => $tx->block_hash_id])->first();
-        $confirmations = $block->confirmations;
+        $last_block = $this->Blocks->find()->select(['height'])->order(['height' => 'desc'])->first();
+        $confirmations = $last_block->height - $block->height + 1;
```

Comment:
This can't always be true for the tx page. If it is in the mempool it will have 0 confirmations. I would do a check to see if the block has 1 confirmation first. If it does then we can do this because it has been confirmed by lbrycrd. If there is a reog the block will be deleted so we do not need to cover that case.

Reply:
Noted, I added a condition !